### PR TITLE
Add V2Ray logger

### PR DESCRIPTION
**Pull request by TRAVUC**

### CODE REVIEW

File: common/log/v2ray/logger.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
+package v2ray
```
Comment (eycorsican) [inline common/log/v2ray/logger.go]: Should follow by a blank line

File: common/log/v2ray/logger.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import (
+	"fmt"
+	"os"
```
Comment (eycorsican) [inline common/log/v2ray/logger.go]: Should follow by a blank line

File: common/log/v2ray/logger.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+		l.level = vlog.Severity_Warning
+	case log.ERROR:
+		l.level = vlog.Severity_Error
+	case log.NONE:
+		l.level = vlog.Severity_Unknown
+	}
+}
+
+func (l *v2rayLogger) Debugf(msg string, args ...interface{}) {
+	l.output(vlog.Severity_Debug, msg, args)
```
Comment (eycorsican) [inline common/log/v2ray/logger.go]: `args` should be unpacked, i.e. `args...`

Comment (TRAVUC) [reply common/log/v2ray/logger.go]: Fixed in new commit

**Comment by eycorsican** [comment]
Thx.

**Pull request merged**